flink-libraries/flink-table/src/main/java/org/apache/calcite/sql/fun/SqlStdOperatorTable.java

```diff
@@ -1039,7 +1039,7 @@ public boolean argumentMustBeScalar(int ordinal) {
 		new SqlCollectionTableOperator("TABLE", SqlModality.RELATION);
 
 	public static final SqlOverlapsOperator OVERLAPS =
```

Comment:
Do you think it is a good idea to separate it into another PR? It might simplify the reviewing process.